notify-send.sh

```diff
@@ -19,6 +19,19 @@
 # Desktop Notifications Specification
 # https://developer.gnome.org/notification-spec/
 
+DIR=`dirname "$0"`
+
+# set -x
+
+run_action_handler () {
+  action=`ps axu | grep "notify-action.sh" | grep -v "grep"`
+  if [[ -z "$action" ]] ; then
+    echo "run_action_handler"
```

Comment:
Was this echo supposed to be left?

Reply:
As of some `set -x`, this should have been removed. I did it